src/components/footer.js

```diff
@@ -0,0 +1,257 @@
+import React from "react"
+import styled from "styled-components"
+import dimensions from "../style/dimensions"
+import { StaticQuery, graphql } from "gatsby"
+
+const FooterContainer = styled.div`
+  background-color: #848484;
+  width: 100%;
+  color: #ffffff;
+`
+
+const FooterMainContent = styled.div`
+  @media (min-width: ${dimensions.maxwidthTablet}px) {
+    display: flex;
+  }
+`
+
+const SocialSection = styled.div`
+  padding-left: 80px;
+  padding-top: 50px;
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    padding-left: 32px;
+  }
+`
+
+const FooterHeader = styled.div`
+  font-weight: bold;
+  font-size: 24px;
+`
+
+const SocialIcons = styled.div`
+  display: flex;
+  padding-top: 15px;
+`
+
+const SocialPlaceholder = styled.div`
+  width: 24px;
+  height: 24px;
+  background-color: #a4a4a4;
+  border-radius: 8px;
+  margin-right: 16px;
+`
+const EmailText = styled.div`
+  text-decoration: underline;
+  font-size: 18px;
+  font-weight: bold;
+  padding-top: 25px;
+`
+
+const LinkSection = styled.div`
+  display: flex;
+  margin-left: 9vw;
+  padding-top: 50px;
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    margin-left: 0px;
+  }
+
+  li {
+    padding-bottom: 47px;
+    font-size: 18px;
+    font-weight: bold;
+  }
+
+  a {
+    text-decoration: none;
+    color: #ffffff;
+  }
+
+  ul {
+    list-style: none;
+  }
+`
+
+const LinkColumn = styled.div`
+  padding-right: 9vw;
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    padding-right: 0;
+  }
+`
+
+const EmailSection = styled.div`
+  margin-left: 32px;
+
+  input {
+    width: 20vw;
+    height: 36px;
+    margin-top: 10px;
+    border: solid #626262 1px;
+  }
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    input {
+      width: 80%;
+      height: 36px;
+      margin-top: 10px;
+      border: solid #626262 1px;
+    }
+  }
+
+  input::placeholder {
+    padding-left: 14px;
+  }
+
+  @media (min-width: ${dimensions.maxwidthTablet}px) {
+    padding-top: 50px;
+    position: absolute;
+    right: 62px;
+  }
+`
+
+const BottomLinks = styled.div`
+  a {
+    text-decoration: none;
+    color: black;
+  }
+  ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    padding-bottom: 30px;
+  }
+
+  li {
+    font-size: 12px;
+    padding-right: 28px;
+  }
+
+  @media (max-width: ${dimensions.maxwidthTablet}px) {
+    ul {
+      display: flex;
+      justify-content: center;
+      list-style: none;
+      padding-top: 80px;
+      padding-bottom: 12px;
+    }
+  }
+`
+
+const AddressSection = styled.div`
```

Comment:
This is a general trend for project organization in Scout rn:
```
component-name/
   component.js
   component-styles.js
   <anything else related to the component>
```
When you get a lot of files this keeps stuff really easy to follow, would you mind organizing this way? Also, could you please keep the styles inside their own folder?